.github/workflows/dev.yaml

```diff
@@ -52,19 +52,24 @@ jobs:
       if: always()
       run: poetry run behave tests/features --tags=-skip --junit
 
-    - name: Upload behave test results from json file
+    - name: Remove broken characters from xml test files
+      if: always()
+      run: |
+        sed -i -e 's/[\x0A\x1B]//g' reports/TESTS-*.xml
```

Comment:
Can you stick in a comment to explain why we need to remove these chars? just mention that it's to do with the later test results extraction step.